External/FEXCore/Source/Interface/Core/JIT/Arm64/AtomicOps.cpp

```diff
@@ -10,7 +10,7 @@ tags: backend|arm64
 namespace FEXCore::CPU {
 using namespace vixl;
 using namespace vixl::aarch64;
-#define DEF_OP(x) void Arm64JITCore::Op_##x(IR::IROp_Header *IROp, IR::NodeID Node)
+#define DEF_OP(x) void Arm64JITCore::Op_##x(IR:: IROp_Header const *IROp, IR::NodeID Node)
```

Comment:
In a few of these, it seems like a space was added after the `IR::` qualifier

Reply:
Weird, not sure how that got there, will fix.

Reply:
Resolved.